clang/lib/CodeGen/SYCLLowerIR/LowerWGScope.cpp

```diff
@@ -380,8 +380,8 @@ static void copyBetweenLocalAndShadow(AllocaInst *L, GlobalVariable *Shadow,
   Type *T = L->getAllocatedType();
 
   if (T->isAggregateType()) {
-    auto LocAlign = L->getAlignment();
-    auto ShdAlign = Shadow->getAlignment();
+    auto LocAlign = MaybeAlign(L->getAlignment());
```

Comment:
LLVM suggests to use `getAlign()` instead of `getAlignment()` now.
See https://github.com/intel/llvm/blob/531c1161b9758aa7eae1a4dd8ecf11fdf4417b29/llvm/include/llvm/IR/Instructions.h#L243
Shouldn't we use it?

Reply:
This is method of `LoadInst`, while `L` here is `AllocaInst`.

I guess this `getAlign` API is not yet ready for all instructions, so the current patch seems correct to me at this point

Reply:
Then we can add some TODO.

Reply:
Added `TODO`